payments-core/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -1110,7 +1110,7 @@ class Stripe internal constructor(
                     apiKey = publishableKey,
                     stripeAccount = stripeAccountId
                 )
-            )
+            ).getOrThrow()
```

Comment:
Complicated change: We can either return `null` _or_ throw but not both, which is a deviation from previous behavior.

With the previous implementation, the parser could return `null` if it was unable to parse the response. Now, we throw an `APIException` for that (or rather, we return `Result.failure(APIException())`).

My proposal is to throw for any failure and tighten the return type from `Source?` to `Source`, given that the merchant already has to handle exceptions.